### DISCUSSION TITLE
Browse Games got stuck

### DISCUSSION THREAD

**Discussion by jeff39389327** [smly/mjai.app#120] [Bug report] [answered]
Batch#AIJ48  6/22
This batch is in progress! (40 of 1000)

**Comment by smly** [comment]
Sorry for the late response!
I finally have the time to address this issue.
There is a lot of work to be done, including identifying bugs and rolling back data.
I expect to have it resolved by today or tomorrow.

Thanks!

**Reply by smly** [reply, marked as answer]
Resolved!
